index.bs

```diff
@@ -751,14 +751,16 @@ A <dfn id=concept-sensor>sensor</dfn> has an associated [=ordered set|set=]
 of <dfn>activated sensor objects</dfn>.
 This set is initially [=set/is empty|empty=].
 
-A [=sensor=] has an associated <dfn>latest reading</dfn> [=ordered map|map=]
-which holds the latest available [=sensor readings=].
+For the current [=browsing context=] a [=sensor=] has an associated <dfn>latest reading</dfn>
+[=ordered map|map=] which holds the latest available [=sensor readings=].
 
-Any time the UA obtains a new [=sensor reading=] for a [=sensor=] from the underlying platform,
-it invokes [=update latest reading=] with the [=sensor=] and the [=sensor reading=] as arguments.
+Note: User agents may share [=sensor readings=] [=ordered map|map=] between different
```

Comment:
s/may/can/

We don't generally use RFC 2119 terms in informative notes for clarity.

Reply:
done